src/cbc_sdk/helpers.py

```diff
@@ -111,7 +111,7 @@ def get_object_by_name_or_id(cb, cls, name_field="name", id=None, name=None, for
     try:
         if id:
             attempted_to_find = "ID of {0:d}".format(id)
-            objs = [cb.select(cls, id, force_init=force_init)]
```

Comment:
As far as I saw NewBaseModel has force_init as parameter of the __init__, but not the classes that are inheriting that base class.